android/source/VideoLocker/res/drawable/switch_track.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@drawable/switch_defaultbg" android:state_focused="true"/>
+    <item android:drawable="@drawable/switch_cyanbg" android:state_checked="true"/>
     <item android:drawable="@drawable/switch_defaultbg" android:state_checked="false"/>
     <item android:drawable="@drawable/switch_cyanbg"/>
```

Comment:
should this have ``state_checked="false"`` to match the other one? Otherwise, :+1: